wordcloud/wordcloud.py

```diff
@@ -729,6 +732,244 @@ def __array__(self):
     def to_html(self):
         raise NotImplementedError("FIXME!!!")
 
+    def to_svg(self, embed_font=False, optimize_embedded_font=True, embed_image=False):
+        """Export to SVG.
+
+        Font is assumed to be available to the SVG reader. Otherwise, text
+        coordinates may produce artifacts when rendered with replacement font.
+        It is also possible to include a subset of the original font in WOFF
+        format using `embed_font` (requires `fontTools`).
+
+        Note that some renderers do not handle glyphs the same way, and may
```

Comment:
But isn't the issue that PIL renders the font differently from some output renderers, and hence the masking for building the word cloud is only approximate? Is there a way to get the renderer, or the embedding to emulate the rendering used by PIL? Or is that getting too crazy?

Reply:
After some investigations, my guess is that some renderers do not properly handle [complex text layouts](https://en.wikipedia.org/wiki/Complex_text_layout). Chrome and Inkscape seem to do this, while Internet Explorer doesn't.

There have been [some discussions on Pillow's repository](https://github.com/python-pillow/Pillow/issues/1089) to support this, which have been solved by using `raqm`. However, this requires many dependencies, and it is not included by default (see [Pillow's installation guide](https://pillow.readthedocs.io/en/stable/installation.html)). You can check if you Pillow installation has it:

```
from PIL import features
print(features.check("raqm"))
```

If it is the case, then the output will be similar to Chrome, which is probably the correct way to render text. If not, then it will use a simpler strategy (i.e. glyphs are rendered independently).

Here is the output of `to_image` when `raqm`is available:

![bar_radm](https://user-images.githubusercontent.com/11242808/71901882-29bc0c80-3161-11ea-8f02-bf5e26f26441.png)

I think we should not assume that `raqm` is available. And since other renderers are not consistent anyway, it does not make sense to force one of those "styles".

Still, to actually answer your question, it might be possible to drop these complex layouts by stripping some information when embedding the font. The `raqm` feature would therefore need to be explicitly disabled.

However, I believe it is a separate issue, as this has a limited impact on the SVG result. The goal of this PR is not to challenge the current design, which has worked well for years. But, we can indeed open an issue to keep track of this.

Reply:
Thanks for this! Great analysis. Could you tweak the note to emphasise that the relationship between the generation and the online rendering is what might make it look awkward? And maybe to say that if raqm is installed, generation will accommodate complex text layouts handled by some renderers.

Reply:
And personally, I think disabling complex layouts is not a good way to go, particularly with respect to Arabic and Indic writing systems, unless we adjust unicode handling to correspond.

Reply:
I have updated the docstring. Indeed, we should avoid hacking into this.